src/main/scala/fs2/pull1.scala

```diff
@@ -51,6 +51,17 @@ private[fs2] trait pull1 {
   def await1Option[F[_],I]: Handle[F,I] => Pull[F,Nothing,Option[Step[I,Handle[F,I]]]] =
     h => h.await1.map(Some(_)) or Pull.pure(None)
 
+  def changing[F[_],I](eqf:(I,I) => Boolean):Handle[F,I] => Pull[F,I,I] = {
```

Comment:
okay, but what is the story on pipe? aka `zipWithPrevious()() pipe collect()` ???